scenarios/v5/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+services:
+
+  conjur-master:
+    image: registry.tld/conjur-appliance:5.0-stable
+    container_name: cdemo_conjur-master
+    ports:
+      - "443:443"
+    security_opt:
+        - "seccomp:unconfined"
+    volumes:
+      - ./seeds:/tmp/seeds
+
+  conjur-standby:
+    image: registry.tld/conjur-appliance:5.0-stable
+    container_name: cdemo_conjur-standby
+    security_opt:
+        - "seccomp:unconfined"
+    links:
+      - conjur-master
+    volumes:
+      - ./seeds:/tmp/seeds
+
+  conjur-follower:
+    image: registry.tld/conjur-appliance:5.0-stable
+    container_name: cdemo_conjur-follower
+    ports:
+      - "8443:443"
+    security_opt:
+        - "seccomp:unconfined"
+    links:
+      - conjur-master
+    volumes:
+      - ./seeds:/tmp/seeds
+
+  conjur-cli:
+    image: conjurinc/cli5
+    container_name: cdemo_conjur-cli
+    environment:
+      CONJUR_APPLIANCE_URL: http://conjur-master
+      CONJUR_ACCOUNT: demo
+      CONJUR_AUTHN_LOGIN: admin
+    volumes:
+      - ./policy:/src/policy
+    links:
+      - conjur-master
```

Comment:
A pattern we're using in other projects is to use docker volumes. This let's us avoid making local directories, but still allows containers to share data (seed files). A volume can be declared with:
```yml
version: '3'

volumes:
  seeds:

services:
...
```
This gives us a named volume that can be attached to containers:
```yml
  conjur-master:
    image: registry.tld/conjur-appliance:5.0-stable
    ...
    volumes:
      - seeds:/tmp/seeds
```
The seed file generation should work as we're operating in the root directory, but it's probably worth updating the seed file export file path to explicitly reference the `/tmp/seeds` directory:
```sh
docker exec cdemo_conjur-master bash -c "evoke seed follower conjur-follower > /tmp/seeds/follower-seed.tar"
```

Reply:
Yeah, and they're secure too. The basic webapp demo uses a shared volume to distribute the API key.

Reply:
not sure how to address this, but one recurring issue is undoing changes one makes to files during the course of doing a demo, and forgetting to undo those changes to revert to the initial demo state. Such as changing !permit to !deny in a policy file. If you forget to change it back, your demo starts up w/ "403 Unauthorized" messages that you have to explain away. Not sure how to address that unless we copied all demo resources into container file systems.

Reply:
That's easy :P 
```
$ git reset --hard HEAD
```
That'll get you back to the last commit you the branch you're working in.

Reply:
Might be worth adding that to the `finish` method if you're making changes.  The alternative is to add a `cleanup.sh` script that runs the above, potentially also with:
```
$ git clean -fd
```
which will remove any newly created files.